sql-join/where-are-they.sql

```diff
@@ -0,0 +1,7 @@
+select "line1",
+      "c"."name",
+      "district",
+      "C"."name"
+from "addresses"
+join "cities" as "c" using ("cityId")
+join "countries" as "C" using ("countryId");
```

Comment:
For clarity, I'd suggest using alias that aren't just different cases of the same letter. Bragging rights for getting the country in there, though.